tests/trigger/test_sqli.py

```diff
@@ -0,0 +1,38 @@
+from vulnpy.trigger import sqli
+from tests.trigger.base_test import BaseTriggerTest
+
+
+class BaseSqliTest(BaseTriggerTest):
+    """All SQLi triggers catch their exceptions"""
+
+    @property
+    def exception_input(self):
+        return "invalid ' sql"
+
+    def test_exception(self):
+        pass
+
+    def test_exception_caught(self):
+        assert self.trigger_func(self.exception_input) == "<>"
+
+    @property
+    def good_input(self):
+        return "d', '4'),('e", "a,3;b,5;c,1;d,4;e,1"
```

Comment:
same question here, can we use the classic attack str? Or maybe there was a great reason you used this data?